Firmware/LoRaSerial_Firmware/Radio.ino

```diff
@@ -772,16 +785,9 @@ void hopChannel()
   radio.clearFHSSInt();
   timeToHop = false;
 
-  if (settings.autoTuneFrequency == true)
-  {
-    if (radioState == RADIO_LINKED_RECEIVING_STANDBY || radioState == RADIO_LINKED_ACK_WAIT
-        || radioState == RADIO_BROADCASTING_RECEIVING_STANDBY) //Only adjust frequency on RX. Not TX.
-      radio.setFrequency(channels[radio.getFHSSChannel()] - frequencyCorrection);
-    else
-      radio.setFrequency(channels[radio.getFHSSChannel()]);
-  }
-  else
-    radio.setFrequency(channels[radio.getFHSSChannel()]);
+  setRadioFrequency(settings.autoTuneFrequency
```

Comment:
Your changes are more succinct but at the cost of decreasing clarity. Please leave as is.